### PR TITLE
[CAMEL-16856] Add azure-storage-blob-changefeed bundle in the azure-storage-blob-feature

### PR DESCRIPTION
**Pull request by jbonofre**
When the bundle will be released, I will update the PR with a property and released version.

### CODE REVIEW

**Review by oscerd — CHANGES_REQUESTED** [review]
Unless it is pushed somewhere wouldn't merge this PR, until the release is out

File: platforms/karaf/features/src/main/resources/features.xml [hunk 1]
```diff
@@ -268,6 +268,7 @@
     <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.azure-core/${azure-core-bundle-version}</bundle>
     <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.azure-storage-common/${azure-storage-common-bundle-version}</bundle>
     <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.azure-storage-blob/${azure-storage-blob-bundle-version}</bundle>
+    <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.azure-storage-blob-changefeed/12.0.0-beta8_1-SNAPSHOT</bundle>
```
Comment (omarsmak) [inline platforms/karaf/features/src/main/resources/features.xml:271]: Should the version moved to the `pom.xml`?

Comment (jbonofre) [reply platforms/karaf/features/src/main/resources/features.xml:271]: Yes, as you can see in the description/comment, I will use a property and correct released version when bundle release is done.

**Comment by jbonofre** [comment]
@oscerd yup, it's just to keep track. I will update the PR and merge only once the bundle release is done.

**Comment by jbonofre** [comment]
So, the change has been done without considering this PR that was in done for that. Thanks @oscerd .

**Pull request closed without merging**